examples/tutorials/hybrid_demucs_tutorial.py

```diff
@@ -0,0 +1,346 @@
+"""
+Hybrid Demucs Music Separation
+==============================
+
+**Author**: `Sean Kim <https://github.com/skim0514>`__
+
+This tutorial shows how to use the Hybrid Demucs model in order to
+perform music separation
+
+"""
+
+
+######################################################################
+# 1. Overview
+# -----------
+# 
+# Performing music separation is composed of the following steps
+# 
+# 1. Build the Hybrid Demucs pipeline.
+# 2. Format the waveform into chunks of expected sizes and loop through
+#    chunks (with overlap) and feed into pipeline.
+# 3. Collect output chunks and combine according to the way they have been
+#    overlapped.
+# 
+# The Hybrid Demucs model is a developed version of the Demucs model, a
+# waveform based model which succesfully separated music into its
+# respective parts. Hybrid Demucs effectively uses spectrogram to learn
+# through the frequency domain and also moves to time convolutions.
+# 
+
+
+######################################################################
+# 2. Preparation
+# --------------
+# 
+# First, we install the necessary dependencies. In addition to
+# ``torchaudio``, ``mir_eval`` is required to perform Si-SDR calculations
+# 
+
+import torch
+import torchaudio
+import typing
+from IPython.display import Audio
+from torchaudio.utils import download_asset
+import matplotlib.pyplot as plt
+import mir_eval
+
+try:
+    import google.colab
+
+    print(
+        """
+        To enable running this notebook in Google Colab, install nightly
+        torch and torchaudio builds and the requisite third party libraries by
+        adding the following code block to the top of the notebook before running it:
+
+        !pip3 uninstall -y torch torchvision torchaudio
+        !pip3 install --pre torch torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cpu
+        !pip3 install mir_eval
+        """
+    )
+except ModuleNotFoundError:
+    pass
+
+print(torch.__version__)
+print(torchaudio.__version__)
```

Comment:
Can we separate this print statements and import torch, torchaudio to separate block?
That way, users running this tutorial on Colab can always see the versions they are running .

See 

https://github.com/pytorch/audio/blob/main/examples/tutorials/audio_io_tutorial.py#L10-L16